menu/navigation.json

```diff
@@ -542,6 +542,10 @@
                     "label": "Choosing an Instance type",
```

Comment:
Maybe we can add a link to the Instances datasheet from within this page?

Reply:
Link added